src/chains/ethereum/ethereum/tests/forking/forking.test.ts

```diff
@@ -825,31 +825,52 @@ describe("forking", function () {
           ); //sanity check
         }
       }
+
+      /*
+        
+        - initialises localProvider as a fork of remoteProvider
+        - sets `value1` to `initialValue` (if it is not null)
+        - create snapshot
+        - iterate `snapshotValues`, setting `value1` to each value
+        - revert
+        - ensure that `value1` has reverted to either `initialValue`, or `initialRemoteValue` if not provided
+      */
```

Comment:
```suggestion

      /**
       *  - Initializes `localProvider` as a fork of `remoteProvider`.
       *  - Sets `value1` to `localInitialValue` (if it is not null).
       *  - Creates a snapshot .
       *  - Iterates `snapshotValues`, setting `value1` to each of those values.
       *  - Reverts.
       *  - Ensures that `value1` has reverted to either `localInitialValue` or
       *    `remoteInitialValue` (if the `localInitialValue` was not provided).
       */
```
Proof read me on this, but I think you kept some of the variable names from before your refactor in this comment. I think this fixes that. I also made it a proper js-doc comment so it displays when you hover over the function name.

Reply:
He also Americanized your words. Sneaky sneaky! :laughing:

Reply:
:zipper_mouth_face: